docs/debug-kernel.md

```diff
@@ -0,0 +1,45 @@
+# Get Clear Containers Kernel logs
+
+Clear Containers runtime launches a special virtual machine when a new pod is
+created. This VM uses a custom kernel inside the VM.
+
+The kernel boot logs are disabled by default. However these logs may be desired
+when the kernel is customized by the end user, or if the virtual machine boot
+fails due to the kernel.
+
+To provide a debug log of the kernel boot, Clear Containers supports an option
+to enable kernel boot logs using the configuration file.
+
+1- Enable debug in configuration.toml
+
+```
+enable_debug = true
+```
+
+2- Run a container to generate the logs
+
+```
+sudo docker run -ti busybox true
+```
+
+3- Filter the kernel boot logs from the `cc-proxy` logs
+The `cc-proxy` logs show the sources of its collated information. To see
+the kernel boot logs, filter `cc-proxy` logs by the QEMU serial console.
+The QEMU serial console is represented by `source=qemu`.
+
+```
+journalctl -u cc-proxy | grep source=qemu
+```
+
+4- Obtaining logs using standalone proxy
+
+In some cases it may be easier to run the proxy standalone by first stopping
+the cc-proxy service and running it standalone to capture the logs
+
+```
+sudo systemctl stop cc-proxy
+sudo /usr/libexec/clear-containers/cc-proxy -log debug`
```

Comment:
extraneous single quote on the end of the line.

Reply:
Fixed